mozilla-pine/setup

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -x # Show commands
+set -eu # Errors/undefined vars are fatal
+set -o pipefail # Check all commands in a pipeline
+
+# See ../mozilla-central/setup for documentation
+# The mozilla-central repo also uploads an updated gecko.tar which
+# includes pine commits, so we don't need our own upload script.
+
+REVISION_TREE=pine
+REVISION_ID=latest
+
+date
+
+source $CONFIG_REPO/shared/resolve-gecko-revs.sh $REVISION_TREE $REVISION_ID
+
+date
+
+$CONFIG_REPO/shared/checkout-gecko-repos.sh $REVISION_TREE "pine" "$INDEXED_HG_REV"
+
+date
+
+$CONFIG_REPO/shared/fetch-tc-artifacts.sh $REVISION_TREE $INDEXED_HG_REV "$PREEXISTING_HG_REV"
```

Comment:
I think you might need to drop this call, unless you plan on enabling all the necessary jobs on the pine branch? Not sure.

Reply:
I enabled searchfox jobs on pine and it seems to have worked, with results showing up on https://treeherder.mozilla.org/jobs?repo=pine today.